CMake/external_pybind11.cmake

```diff
@@ -84,3 +81,19 @@ endfunction()
 
 # Trigger the build
 get_pybind11()
+
+# This function override "pybind11_add_module" function,  arguments is same as "pybind11_add_module" arguments
+# pybind11_add_module(<name> SHARED [file, file2, ...] )
+# Must be declared after pybind11 configuration above
+# "_pybind11_add_module" is calling the origin pybind11 function
+function( pybind11_add_module _arg_project_name _arg_library_type ...)
```

Comment:
I'd remove the `_arg_` from the names... why are they needed?

Reply:
Nothing special,  will remove